src/api/volunteer-event/index.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  RegionOptions,
+  VolunteerEventCategory
+} from '@/constants/volunteerEvent';
+import api from '../instance';
+import { EventStatus, HomeVolunteerEvent } from '@/types/volunteerEvent';
+
+export type HomeEventFilter = {
+  category?: 'all' | VolunteerEventCategory;
+  status?: 'all' | EventStatus;
+  longitude?: number;
+  latitude?: number;
+  address?: '내 주변' | RegionOptions;
+  isFavorite?: boolean;
+};
+
+export const queryKey = {
+  all: 'all-volunteer-event' as const,
```

Comment:
```suggestion
  all: ['volunteer-event', 'all'] as const,
```

하영님
저희가 봉사활동 신청/신청철회시에
홈리스트/보호소리스트/어드민리스트 3개 리스트를 동기화 해야 하는데 한번에 refetch 하기 위해서 쿼리키 계층구조를 같이 가져가야한다고 하네요!

'volunteer-event' 상위키로 가져가도록 변경해주시면 감사드리겠습니다앗

[쿼리키 매칭 로직](https://tanstack.com/query/v4/docs/react/guides/query-invalidation#query-matching-with-invalidatequeries)